src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-container.component.ts

```diff
@@ -396,7 +408,7 @@ export class CodeEditorTutorAssessmentContainerComponent implements OnInit, OnDe
     }
 
     private calculateTotalScore() {
-        const feedbacks = [...this.referencedFeedback, ...this.unreferencedFeedback];
+        const feedbacks = [...this.referencedFeedback, ...this.unreferencedFeedback, ...this.automaticFeedback];
```

Comment:
Unfortunately its not that easy 😅. With the extended score calculation with bonus points, sca penalties and max penalties we cannot just add the feedback credits to get the overall points. Also the score is in percent. I think we should let the server do the calculation here.

Reply:
We discussed this and this is fine for now. We will address this in a later PR.